lib/src/client.rs

```diff
@@ -260,7 +264,8 @@ impl ClientInner {
                     config.zkp.prover_active,
                     #[cfg(feature = "zkp-prover")]
                     None,
-                    config.zkp.setup_keys_path,
```

Comment:
Maybe we should use the opportunity to get rid of the conditional arguments.
Instead, let's create two functions with different names.